rest-api-spec/src/main/resources/rest-api-spec/test/mget/80_deprecated.yml

```diff
@@ -0,0 +1,64 @@
+
+---
+"Deprecated parameters should produce warning in Multi Get query":
+# MODIFY in 7.x as these throw errors instead of warnings
+
+   - skip:
+       version: " - 6.99.99"
+       reason:  some parameters are deprecated starting from 7.0, their equivalents without underscore are used instead
+       features: "warnings"
+
+   - do:
+       index:
+           index:  test_1
+           type:   test
+           id:     1
+           body:   { foo: bar }
+
+   - do:
+       index:
+           index:  test_1
+           type:   test
+           id:     2
+           body:   { foo: baz }
+
+   - do:
+       mget:
+           body:
+               docs:
+                   - { _index: test_1, _type: test, _id: 1, _routing : test1 }
+                   - { _index: test_1, _type: test, _id: 2, _routing : test1 }
+       warnings:
+           - "Deprecated field [_routing] used, expected [routing] instead"
+
+---
+"Deprecated parameters should produce warning in Multi Get query2":
+# MODIFY in 7.x as these throw errors instead of warnings
```

Comment:
In master these should still work, right? Until we make a followup that removes support for the camelCase and stuff.